recipes/example/meta.yaml

```diff
@@ -9,7 +11,7 @@ source:
     fn: simplejson-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/s/simplejson/simplejson-{{ version }}.tar.gz
```

Comment:
Do we want to start templating the name too? I've found this handy in a few cases and other people seem to have picked up on it. Though this suggestion may be out of scope for this proposal.